src/pages/Invoice/CreateInvoice.js

```diff
@@ -1,7 +1,376 @@
+                                </div>
+                                <div className="input-group px-2">
+                                    <label htmlFor="referenceNumber">
+                                        <FormattedMessage id="invoice.reference.number"></FormattedMessage>{" "}
+                                    </label>
+                                    <i className="fa fa-hashtag"></i>
+                                    <input
+                                        className="input-sm"
+                                        type="text"
+                                        name="referenceNumber"
+                                        required
+                                        onChange={(e) =>
+                                            setInvoiceRecipentDetails({
+                                                ...invoiceRecipentDetails,
+                                                referenceNumber: e.target.value,
+                                            })
+                                        }
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div className="invoice_items py-5 mx-5">
+                        <table className="table-bordered">
+                            <thead>
+                                <tr>
+                                    <th>
+                                        {" "}
+                                        <FormattedMessage id="title.items"></FormattedMessage>{" "}
+                                    </th>
+                                    <th>
+                                        {" "}
+                                        <FormattedMessage id="invoice.quantity"></FormattedMessage>{" "}
+                                    </th>
+                                    <th>
+                                        {" "}
+                                        <FormattedMessage id="item.price"></FormattedMessage>{" "}
+                                    </th>
+                                    <th>
+                                        {" "}
+                                        <FormattedMessage id="invoice.amount"></FormattedMessage>{" "}
+                                    </th>
+                                    <th className="table-action"></th>
+                                </tr>
+                            </thead>
+                            {invoiceRecipentDetails.items.length > 0 && (
+                                <tbody>
+                                    {invoiceRecipentDetails.items.map(
+                                        (item, idx) => (
+                                            <tr key={idx}>
+                                                <td> {item.name} </td>
+                                                <td>{item.amount}</td>
```

Comment:
it should be `quantity` & input field can configure by user

Reply:
Fixed this